dnd5eApi/models/feats.py

```diff
@@ -0,0 +1,21 @@
+from dnd5eApi import db
+#from dnd5eApi.models.racial_languages import racial_languages
+
+class Feats( db.Model ):
+  __tablename__ = "feats"
+
+  id = db.Column( db.Integer, primary_key = True )
+  name = db.Column( db.String, nullable = False )
+  description = db.Column( db.String, nullable = False )
+  requirements = db.Column( db.String, nullable = False )
+  bonuses = db.Column( db.String, nullable = False )
+  #races = db.relationship( 'Race', secondary = racial_languages, backref = db.backref( 'language', lazy = 'dynamic' ) )
```

Comment:
It is a placeholder for now.  I didn't see a reference for this in your ruby code, so I figured you had not developed the linker table yet.